gcp-test/tests/gcp_compute_global_address/variables.tf

```diff
@@ -36,11 +36,25 @@ data "null_data_source" "resource" {
   }
 }
 
+resource "google_compute_network" "network" {
+  project      = var.gcp_project
+  provider      = google-beta
+  name          = var.resource_name
+  auto_create_subnetworks = false
+}
+
 resource "google_compute_global_address" "named_test_resource" {
-  name        = var.resource_name
-  description = "Test global address to validate integration test."
+  provider      = google-beta
+  project       = var.gcp_project
```

Comment:
Why can't we create the basic Global address and run the test?
```
resource "google_compute_global_address" "default" {
  name = "global-appserver-ip"
}
```

Reply:
We need a `PREMIUM` network for global address otherwise it is failing the integration test for `STANDARD` one

Reply:
Here is the error details from TF while creating the resource 
`Error: Error creating GlobalAddress: googleapi: Error 400: STANDARD network tier (the project's default network tier) is not supported: STANDARD network tier is not supported for global address., badRequest`